File: app/src/main/java/club/hutcwp/coolweather/bean/weather/AQI.java
/*
 * Copyright (c) 2017. The Android Project Created By Hutcwp.
 */

package club.hutcwp.coolweather.bean.weather;

/**
 * Created by hutcwp on 2017/4/9.
 * Mail : hutcwp@foxmail.com
 * Blog : hutcwp.club
 * GitHub : github.com/hutcwp
 */

public class AQI {


    /**
     * city : {"aqi":"42","pm10":"42","pm25":"29","qlty":"优"}
     */

    private CityBean city;

    public CityBean getCity() {
        return city;
    }

    public void setCity(CityBean city) {
        this.city = city;
    }

    public static class CityBean {
        /**
         * aqi : 42
         * pm10 : 42
         * pm25 : 29
         * qlty : 优
         */

        private String aqi;
        private String pm10;
        private String pm25;
        private String qlty;

        public String getAqi() {
            return aqi;
        }

        public void setAqi(String aqi) {
            this.aqi = aqi;
        }

        public String getPm10() {
            return pm10;
        }

        public void setPm10(String pm10) {
            this.pm10 = pm10;
        }

        public String getPm25() {
            return pm25;
        }

        public void setPm25(String pm25) {
            this.pm25 = pm25;
        }

        public String getQlty() {
            return qlty;
        }

        public void setQlty(String qlty) {
            this.qlty = qlty;
        }
    }
}

File: app/src/main/java/club/hutcwp/coolweather/bean/weather/AQI.kt
/*
 * Copyright (c) 2017. The Android Project Created By Hutcwp.
 */

package club.hutcwp.coolweather.bean.weather

/**
 * Created by hutcwp on 2017/4/9.
 * Mail : hutcwp@foxmail.com
 * Blog : hutcwp.club
 * GitHub : github.com/hutcwp
 */

class AQI {


    /**
     * city : {"aqi":"42","pm10":"42","pm25":"29","qlty":"优"}
     */

    var city: CityBean? = null

    class CityBean {
        /**
         * aqi : 42
         * pm10 : 42
         * pm25 : 29
         * qlty : 优
         */

        var aqi: String? = null
        var pm10: String? = null
        var pm25: String? = null
        var qlty: String? = null

        override fun toString(): String {
            return "CityBean(aqi=$aqi, pm10=$pm10, pm25=$pm25, qlty=$qlty)"
        }
    }

    override fun toString(): String {
        return "AQI(city=$city)"
    }

}
